src/toolkit/matl_buy_policy.h

```diff
@@ -191,17 +207,22 @@ class MatlBuyPolicy : public Trader {
   };
 
   void set_manager(Agent* m);
+  void set_total_inv_tracker(TotalInvTracker* t);
+  void set_inv_policy(std::string p, double x, 
+                      double y = std::numeric_limits<double>::max());
   /// requires buf_ already set
   void set_fill_to(double x);
   /// requires buf_ already set
-  void set_req_when_under(double x);
+  void set_req_at(double x);
+  void set_ccap(double x);
   void set_quantize(double x);
   void set_throughput(double x);
   void init_active_dormant();
 
   ResBuf<Material>* buf_;
-  std::string name_;
-  double fill_to_, req_when_under_, quantize_, throughput_;
+  TotalInvTracker* buf_tracker_;
+  std::string name_, inv_policy;
+  double fill_to_, req_at_, quantize_, throughput_, ccap_, cycle_total_inv_;
```

Comment:
Is there a substantive difference between `req_at_` and `req_when_under_`?

Reply:
Also, can we use a longer name for `ccap_` - perhaps `cumulative_capacity_`? Long variable names can really help readability, esp. for new developers and me in 6 months.

Reply:
No, I should probably change everything to `req_at` because that's the more precise term (the other one is the previously-existing term)